### DISCUSSION TITLE
KeyError: 'login'

### DISCUSSION THREAD

**Discussion by Hybert404** [TwitchIO/TwitchIO#436] [Questions]
````
import twitchio

oauth_token = "1234"

async def event_message(ctx):
    print(f"Received message: {ctx.content}")

if __name__ == "__main__":
    bot = twitchio.Client(token=oauth_token)
    bot.event(event_message)
    bot.run()
````

    
I simply tried to read chat messages. I requested oAuth token, put it in the script and it always give me error KeyError: 'login'.
I have tried different twitchio versions and requesting new oAuth. Still doesn't work. Any help?

**Comment by chillymosh** [comment]
@Hybert404 you are using an app token, you need to use a user token.

I'm not sure why you have bot.event() in your if name main block
